addons/stock/models/stock_orderpoint.py

```diff
@@ -309,8 +309,10 @@ def _get_orderpoint_action(self):
         to_refill = defaultdict(float)
         all_product_ids = []
         all_warehouse_ids = []
+        ConfigParameter = self.env['ir.config_parameter'].sudo()
+        forecast_forecast_interval = ConfigParameter.get_param("stock.forecast_interval", "3")
```

Comment:
May I suggest renaming this to something that suggests the unit we're using? Perhaps `forecast_months` or `forecast_interval_months`?